core/benches/scheduler.rs

```diff
@@ -0,0 +1,494 @@
+use {
+    criterion::{black_box, criterion_group, criterion_main, Criterion, Throughput},
+    crossbeam_channel::{unbounded, Receiver, Sender},
+    jemallocator::Jemalloc,
+    solana_core::banking_stage::{
+        scheduler_messages::{ConsumeWork, FinishedConsumeWork, MaxAge},
+        transaction_scheduler::{
+            greedy_scheduler::{GreedyScheduler, GreedySchedulerConfig},
+            prio_graph_scheduler::{PrioGraphScheduler, PrioGraphSchedulerConfig},
+            scheduler::{PreLockFilterAction, Scheduler},
+            transaction_state::TransactionState,
+            transaction_state_container::{StateContainer, TransactionStateContainer},
+        },
+        TOTAL_BUFFERED_PACKETS,
+    },
+    solana_runtime_transaction::{
+        runtime_transaction::RuntimeTransaction, transaction_with_meta::TransactionWithMeta,
+    },
+    solana_sdk::{
+        compute_budget::ComputeBudgetInstruction,
+        hash::Hash,
+        instruction::Instruction,
+        message::Message,
+        pubkey::Pubkey,
+        signature::Keypair,
+        signer::Signer,
+        system_instruction,
+        transaction::{SanitizedTransaction, Transaction},
+    },
+    std::{
+        sync::{
+            atomic::{AtomicUsize, Ordering},
+            Arc,
+        },
+        time::{Duration, Instant},
+    },
+};
+
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
+
+// A non-contend low-prio tx, aka Tracer, is tag with this requested_loaded_accounts_data_size_limit
+const TAG_NUMBER: u32 = 1234;
+
+fn is_tracer<Tx: TransactionWithMeta + Send + Sync + 'static>(tx: &Tx) -> bool {
+    matches!(
+        tx.compute_budget_instruction_details()
+            .requested_loaded_accounts_data_size_limit(),
+        Some(TAG_NUMBER)
+    )
+}
```

Comment:
since we're generating the txs and not actually receiving. instead of making this new fn `requested_loaded_accounts_data_size_limit`, which requires us to add an additional instruction to our txs, can we just have a specific `Signature` that you overwrite in the tracer, or a specific `Pubkey`.

Reply:
+1, addresses prev comment too!

Reply:
a0f49e4886c83ee12ae1ef677ac32231b1f170bb